WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift

```diff
@@ -19,11 +19,9 @@ struct LoginEpilogueUserInfo {
     }
 
     init(account: WPAccount, loginFields: LoginFields) {
-        email = loginFields.emailAddress
+        self.init(account: account)
```

Comment:
This doesn't work for Google account - the WPAccount does not have an email address and thus crashes on `email = account.email`. That's why, when it's a Google account, it was using `loginFields.emailAddress`. To note, the only thing a Google account needs the WPAccount for is the `account.username`. (And that's actually the only value in a Google WPAccount.)

Reply:
Damn, I thought this was a clever optimization 😜